### PR TITLE
Make using sysctls a task instead of a concept

### PR DESCRIPTION
**Pull request by tengqm**
Closes: #4505

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6808)

### CODE REVIEW

File: docs/tasks/administer-cluster/sysctl-cluster.md [hunk 1]
```diff
@@ -82,28 +58,54 @@ scheduled, but will fail to launch.
...
 ```
+
 Only _namespaced_ sysctls can be enabled this way.
 
+
 ## Setting Sysctls for a Pod
 
-The sysctl feature is an alpha API in Kubernetes 1.4. Therefore, sysctls are set
-using annotations on pods. They apply to all containers in the same pod.
+A number of sysctls are _namespaced_ in today's Linux kernels. This means that
```
Comment (heckj) [inline docs/tasks/administer-cluster/sysctl-cluster.md]: Generally reads great, but it looks like there's some lingering errors in the template - missing segments. Scan to the bottom of https://deploy-preview-6808--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/sysctl-cluster/ 
 - there's a `ERROR: You must define a prerequisites block` and a `ERROR: You must define a steps block`

**Comment by heckj** [comment]
@tengqm - you've got a rebase that needs to happen, and the template isn't rendering anything that I could see to https://deploy-preview-6808--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/sysctl-cluster/ - would you double check your updates?

**Comment by heckj** [comment]
@tengqm am I just missing this? When I navigate to https://deploy-preview-6808--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/sysctl-cluster/, I'm not seeing the content you have moved/updated above. Are you able to see it? Or am I looking at the wrong deployment preview?

**Comment by tengqm** [comment]
@heckj that ... is weird, investigating ... thanks for noticing this.

**Pull request merged**